designer/client/.eslintignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
 src/brace/
-test/
+/test/
```

Comment:
Why do we need this change?

Reply:
intention of this line was to filter out `client/test` but not `client/src/test`. this mistake costed us lots of poor lines in `src/test`